resources/quality/eryone_er20/eryone_er20_high.inst.cfg

```diff
@@ -0,0 +1,79 @@
+[general]
+version = 4
+name = High
+definition = eryone_er20
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = high
+weight = 1
+global_quality = True
+
+[values]
+acceleration_enabled = True
```

Comment:
Why not add this setting to the machine definition? It seems to always be the same. There are quite a few other settings where this is also the case. 

This makes it a lot easier to maintain the profile in the future; since that way the quality profile only has the settings that it actually changes.

Reply:
Hi, nallath
Thanks for your suggestions! You are correct.
Later I will update it. 
Greeting